### ISSUE TITLE
This site can't be reached after installation aaPanel

### ISSUE THREAD

**Issue by anggi117** [aaPanel/aaPanel#88]
Hii aaPanel team
im installing aaPanel for first time in Google cloud, but site can't reached after success installation
![image](https://user-images.githubusercontent.com/42998655/136894380-38ef370a-4533-4658-8619-a65a92028aee.png)
![image](https://user-images.githubusercontent.com/42998655/136894392-876e0fa6-8e5a-40f0-8c1a-659b6fb132de.png)

**Comment by padddi** [comment]
Yes, I can confirm that. Also tried to install aapanel on my server (Ubuntu 20.04.1). The script went through flawlessly. But when I want to open the panel in the web browser, I get the message I mentioned before.

ERR_CONNECTION_REFUSED

Is there already a solution?

**Comment by marcelloinfoweb** [comment]
Same thing with me.
